### DISCUSSION TITLE
OpenFOAM data to EZyRB

### DISCUSSION THREAD

**Discussion by fab6** [mathLab/EZyRB#234] [Q&A] [answered]
Hi,
I just saw EZyRB, it looks quite interesting for POD-ROM approaches. And it seems, that it was used with openfoam before. Can anyone point me to an example using openfoam data? I am having trouble to understand, how I get the data from openfoam into npy format or into EZyRB.
Maybe it is something simple...
Thank you in advance!
Best
Fab

**Comment by fab6** [comment]
I found an option by using fluidfoam
```
import numpy as np
from fluidfoam import readscalar

sol = './'
timename = '0.001'

p = readscalar(sol, timename, 'p')
np.save("pressure.npy", p)
```
Maybe there is an better option...
Best
Fab

**Comment by mtezzele** [comment, marked as answer]
@fab6 you can also have a look at the [Smithers](https://github.com/mathLab/Smithers) package, where we moved all the I/O operations.

**Comment by fab6** [comment]
ah, ok, I see now the io options, thank you!
